packages/ag-charts-website/src/content/docs/accessibility-test/_examples/opening-context-menu-second-chart/alert.ts

```diff
@@ -0,0 +1,20 @@
+// Less obtrusive alert using HTML Popover API.
+(() => {
```

Comment:
Is the IIFE needed?

Reply:
Yes unfortunately, but mostly because TS validation fails without it (`nx validate-examples`). The variable is declared at the global scope, and then complains that all the examples clash with one another :-(